docs/dev/error-messages.md

```diff
@@ -172,6 +172,22 @@ This error has a few potential causes:
 - Make sure you're launching an appropriate number of jobs for your account.
 - If you see this error with iOS Simulator tests, please make sure the timeouts in your test runner/framework are set to a sufficient duration to allow iOS Simulator tests to start up. We recommend a minimum of 2 minutes.
 
+### The New Session Request Redirect not Followed Timeout
+
+**Description**
+
+Your tests start took longer to assign a Sauce Labs Virtual Machine but your test runner did not follow new session redirect before timeout.
```

Comment:
```suggestion
Your test session was abandoned because it took longer than 45 seconds to assign a Sauce Labs Virtual Machine, and your test runner did not follow the new session redirect before timeout.
```